.vscode/settings.json

```diff
@@ -10,5 +10,8 @@
     "eslint.workingDirectories": [".", "./webview-ui"],
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true,
-    "cloudcode.azureExplorerVisible": true
+    "cloudcode.azureExplorerVisible": true,
+    "[typescript]": {
```

Comment:
I think this might conflict with `editor.defaultFormatter` above. Was the prettier formatter not working correctly?